src/client/internal.ts

```diff
@@ -84,3 +84,13 @@ export interface NotificationActionEventTransport {
     controlSource?: 'buttons';  // Additional sources will be added in future release
     controlIndex?: number;      // The index of the originating control, within notification[controlSource]
 }
+
+export interface OpenFinChannelConnectionEvent {
```

Comment:
The type definition is: `onChannelDisconnect(listener: (...args: any[]) => void): Promise<void>;`
I will submit a PR to runtime to add types later.

Reply:
👍

Cool. For now, would move into `connection.ts`, unexported since anywhere else using this is doing something wrong. And consider extending `WindowEvent<,>`, which will give you the last 4 fields here